flight/targets/sparky/fw/pios_board.c

```diff
@@ -167,12 +168,6 @@ static const struct pios_hmc5883_cfg pios_hmc5883_external_cfg = {
 };
 #endif /* PIOS_INCLUDE_HMC5883 */
 
-/* One slot per selectable receiver group.
- *  eg. PWM, PPM, GCS, SPEKTRUM1, SPEKTRUM2, SBUS
- * NOTE: No slot in this map for NONE.
- */
-uintptr_t pios_rcvr_group_map[MANUALCONTROLSETTINGS_CHANNELGROUPS_NONE];
-
 #define PIOS_COM_TELEM_RF_RX_BUF_LEN 512
```

Comment:
so a lot of these constants can be deleted, actually. See https://github.com/TauLabs/TauLabs/pull/1759/files#diff-fddc7ccc67186b59e95c2e594ed8468b for an example on Sparky2.